.obs/dockerfile/slem4r-base-iso/Dockerfile

```diff
@@ -1,13 +1,15 @@
 # SPDX-License-Identifier: Apache-2.0
-#!BuildTag: suse/sle-micro-rancher-iso/base-%%SLEMICRO_VERSION%%:latest
+#!BuildTag: suse/sle-micro-iso/base-%%SLEMICRO_VERSION%%:latest
+#!BuildTag: suse/sle-micro-iso/base-%%SLEMICRO_VERSION%%:%VERSION%
+#!BuildTag: suse/sle-micro-iso/base-%%SLEMICRO_VERSION%%:%VERSION%-%RELEASE%
 #!BuildConstraint: hardware:disk:size unit=G 10
 #!BuildConstraint: hardware:memory:size unit=G 16
 
 ARG SLEMICRO_VERSION
 ARG SLE_VERSION
 
-FROM suse/sle-micro-rancher/base-${SLEMICRO_VERSION}:latest AS os
-FROM suse/sle-micro-rancher/${SLEMICRO_VERSION}:latest AS builder
+FROM suse/sle-micro/base-${SLEMICRO_VERSION}:latest AS os
+FROM suse/sle-micro/${SLEMICRO_VERSION}:latest AS builder
```

Comment:
renaming images to `suse/sle-micro` so we will not change the registry naming again on later releases.

@kkaempf as far as I understood this is what we agreed in Nue